tests/e2e/workflow/version_test.go

```diff
@@ -41,42 +41,27 @@ var _ = Describe("NetworkAddonsConfig", func() {
 		})
 
 		It("should keep reported versions while being changed", func() {
-			done := make(chan bool)
-
-			configSpec := opv1alpha1.NetworkAddonsConfigSpec{
-				Multus:      &opv1alpha1.Multus{},
-				LinuxBridge: &opv1alpha1.LinuxBridge{},
+			versionRemainsTheSame := func() {
+				CheckConfigVersions(operatorVersion, operatorVersion, operatorVersion, CheckImmediately, CheckDoNotRepeat)
 			}
 
-			// Run goroutine that will run update of the config on background,
-			// this is needed so we can continously verify reported versions throughout
-			// the setup.
-			go func() {
```

Comment:
We can remove this goroutine all together we just need a goroutine with the validator and two messages send in the two timers.